package.json

```diff
@@ -14,13 +14,23 @@
     "@fortawesome/free-brands-svg-icons": "^5.13.0",
     "@fortawesome/free-solid-svg-icons": "^5.13.0",
     "@fortawesome/vue-fontawesome": "^0.1.9",
+    "@popperjs/core": "^2.3.3",
+    "@vue/cli-plugin-babel": "~4.2.0",
+    "@vue/cli-plugin-eslint": "~4.2.0",
+    "@vue/cli-plugin-router": "~4.2.0",
+    "@vue/cli-plugin-vuex": "~4.2.0",
+    "@vue/cli-service": "~4.2.0",
+    "@vue/eslint-config-prettier": "^6.0.0",
+    "babel-eslint": "^10.0.3",
     "core-js": "^3.6.4",
     "express": "^4.17.1",
     "jspdf": "^1.5.3",
     "jspdf-autotable": "^3.5.3",
     "leancloud-storage": "^4.5.3",
     "leanengine": "^3.6.0",
     "vue": "^2.6.11",
+    "vue-js-modal": "^1.3.35",
+    "vue-js-popover": "^1.1.8",
```

Comment:
And this?

Reply:
I'm using the vue-js-modal only, so you can delete the vue-js-popover, and keep the vue-js-modal